static/img/gitlab.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we use a monochrome SVG? The existing ones are all monochrome, I think. This would be the first colored one, which could make it stick out awkwardly.

Reply:
Seems like GitLab does not have an official monochrome SVG. See https://about.gitlab.com/press/

Reply:
this is okay then!